README.md

```diff
@@ -2,8 +2,18 @@
 
 A teeny tiny template to start a documentation.
 
+## Deploy
+
 It defaults to deploy to [GitHub Pages](https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site)
 
+1. fork [ZEISS/docs-template](https://github.com/ZEISS/docs-template)
```

Comment:
👋 super minor thing. Can we start with upper letter for the list to have it consistent?